roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -35,6 +35,9 @@ spec:
     - --etcd-certfile={{ etcd_cert_dir }}/node-{{ inventory_hostname }}.pem
     - --etcd-keyfile={{ etcd_cert_dir }}/node-{{ inventory_hostname }}-key.pem
     - --insecure-bind-address={{ kube_apiserver_insecure_bind_address }}
+{% if kube_apiserver_bind_address is defined  %}
+    - --bind-address={{kube_apiserver_bind_address}}
```

Comment:
the kubeadm case should be updated as well IIUC, see insecure-bind-address in roles/kubernetes/master/templates/kubeadm-config.yaml.j2

Reply:
so LGTM, just with this comment addressed

Reply:
Hmm, I'm not sure about this - I deliberately did not update kube_apiserver_insecure_bind_address as I assumed this was e.g an unsecured endpoint used for internal communication, thus should only normally be bound to localhost?

Happy to update if you disagree, but I only wanted to change the bind port for the external-facing API connections.